uber/site_sections/registration.py

```diff
@@ -141,8 +129,12 @@ def change_badge(self, session, id, message='', **params):
                 message = check(attendee)
 
             if not message:
-                message = session.change_badge(attendee, params['badge_type'], params.get('newnum') or 0)
-                raise HTTPRedirect('form?id={}&message={}', attendee.id, message)
+                badge_num = int(params.get('newnum') or 0)
+                if c.AT_THE_CON and badge_num == 0:  # sometimes admins need to unset accidental badge assignments
+                    attendee.badge_num = 0
+                else:
+                    message = session.change_badge(attendee, params['badge_type'], new_num)
+                raise HTTPRedirect('form?id={}&message={}', attendee.id, message or '')
```

Comment:
Semi-related change: while testing I found myself wanting to say, "Hmm, let's unset that badge now that it's been assigned" and I couldn't do it.  This should be possible IMO even if through a hacky approach like forcing someone to enter "0" as the badge number when at-the-con mode is on.